open-webui/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: open-webui_web_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
+
+  ollama:
+    image: ollama/ollama:0.5.4@sha256:18bfb1d605604fd53dcad20d0556df4c781e560ebebcd923454d627c994a0e37
+    volumes:
+      - ${APP_DATA_DIR}/data/ollama:/root/.ollama
+    # We could uncomment this if we want to expose the ollama port to the host for other external apps/programs to use. 
+    # ports:
+    #   - 11434:11434
+    restart: on-failure
+
+  web:
+    image: ghcr.io/open-webui/open-webui:v0.4.8@sha256:c1e4f0927fb0acd53bcbd8bbc92ecaf7ca36a23ee4cb8f25ce90c541012a473a
+    volumes:
+      - ${APP_DATA_DIR}/data/open-webui:/app/backend/data
+    depends_on:
+      - ollama
+    environment:
+      OLLAMA_BASE_URL: "http://open-webui_ollama_1:11434"
```

Comment:
I've changed this to use the full container name of the ollama instance: `open-webui_ollama_1`.

The reason is that if we kept it as just `ollama` and then a user has another container named `ollama` running (e.g., as a standalone app or as part of another app's compose file) then Docker will "helpfully" try and resolve the correct container by just guessing haha. So sometimes it would connect and other times it wouldn't.